src-tauri/tauri.linux.conf.json

```diff
@@ -14,8 +14,15 @@
   },
   "bundle": {
     "linux": {
+      "categories": [
```

Comment:
Is there any documentation about this config property? I don't see it listed in the Tauri config schema: https://schema.tauri.app/config/2